kubectl-fdb/cmd/profile_analyzer.go

```diff
@@ -0,0 +1,111 @@
+/*
+ * profile_analyzer.go
+ *
+ * This source file is part of the FoundationDB open source project
+ *
+ * Copyright 2021 Apple Inc. and the FoundationDB project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"github.com/FoundationDB/fdb-kubernetes-operator/internal"
+	"github.com/spf13/cobra"
+	appsv1 "k8s.io/api/apps/v1"
+	k8serrors "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+func newProfileAnalyzerCmd(streams genericclioptions.IOStreams) *cobra.Command {
+	o := newFDBOptions(streams)
+
+	cmd := &cobra.Command{
+		Use: "analyze-profile",
+		Short: "Analyze FDB shards to find the busiest team",
+		Long: "Analyze FDB shards to find the busiest team",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			clusterName, err := cmd.Root().Flags().GetString("cluster")
+			if err != nil {
+				return err
+			}
+			flags := cmd.Root().Flags().Args()
```

Comment:
What is the benefit of this command vs, directly using the `args` from the cobra command?